src/main/py/peer/peer.py

```diff
@@ -222,21 +226,58 @@ async def send_trade_response(
     async def handle_accept_trade(self, response: OfferResponse):
         """Handle an accepted trade"""
 
-        self.logger.info(response)
+        trade_confirmation = TradeConfirmation(
+            response.trade_id,
+            self.inventory.pending_trades[response.trade_id].get_artwork_key(),
+            self.keys["public"],
+            True,
```

Comment:
For readability in languages that support named parameters, I usually will write it in that fashion: `accepted = true` in this case. In those that don't, I go to the trouble of creating a small `enum`: values like `ACCEPTED` and `REJECTED` in a case like this.